backend/bamboo/models.py

```diff
@@ -0,0 +1,281 @@
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    site = relationship('Site', back_populates='speaker_forms')
+
+
+class Talk(db.Model):
+    __tablename__ = 'talk'
+    id = Column(Integer, primary_key=True)
+    title = Column(String)
+    content = Column(String)
+    video_url = Column(String)
+    slides_url = Column(String)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    slides_id = Column(Integer, ForeignKey('media.id'))
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    slides = relationship('Media', foreign_keys=[slides_id])
+    site = relationship('Site', back_populates='talks')
+    schedule_items = relationship('ScheduleItem', back_populates='talk')
+    talk_categories = relationship('TalkCategory', back_populates='talk')
+
+
+class Category(db.Model):
+    __tablename__ = 'category'
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+
+    talk_categories = relationship('TalkCategory', back_populates='category')
+
+
+class TalkCategory(db.Model):
+    __tablename__ = 'talk_category'
+    id = Column(Integer, primary_key=True)
+    talk_id = Column(Integer, ForeignKey('talk.id'))
+    category_id = Column(Integer, ForeignKey('category.id'))
+
+    talk = relationship('Talk', back_populates='talk_categories')
+    category = relationship('Category', back_populates='talk_categories')
+
+
+class Staff(db.Model):
+    __tablename__ = 'staff'
+    id = Column(Integer, primary_key=True)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    city_id = Column(Integer, ForeignKey('city.id'))
+    staff_id = Column(Integer, ForeignKey('user.id'))
+
+    city = relationship('City', back_populates='staffs')
+    staff = relationship('User', foreign_keys=[staff_id])
+
+
+class City(db.Model):
+    __tablename__ = 'city'
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    address = Column(String)
+    latitude = Column(Float)
+    longitude = Column(Float)
+    start = Column(DateTime)
+    end = Column(DateTime)
+    registration_url = Column(String)
+    live_urls = Column(String)
+    created_at = Column(DateTime, default=datetime.utcnow)
+    updated_at = Column(DateTime, default=datetime.utcnow, onupdate=datetime.utcnow)
+    site_id = Column(Integer, ForeignKey('site.id'))
+
+    staffs = relationship('Staff', back_populates='city')
+    partnerships = relationship('Partership', back_populates='city')
+    venues = relationship('Venue', back_populates='city')
+
+
+class Partership(db.Model):
+    __tablename__ = 'partnership'
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+    url = Column(String)
```

Comment:
It shouldn't have these properties, they belong to `Organization`, it is a relationship table with extra info(`category`)

Reply:
Removed.